lrpar/src/lib/ctbuilder.rs

```diff
@@ -173,6 +233,29 @@ where
         self
     }
 
+    /// If set to true, `process_file_in_src` will return an error if the given grammar contains
+    /// any Shift/Reduce or Reduce/Reduce conflicts. Defaults to `true`.
+    pub fn error_on_conflicts(mut self, b: bool) -> Self {
+        self.error_on_conflicts = b;
+        self
+    }
+
+    /// Returns the grammar conflicts found during generation of the parser, or `None` if there
```

Comment:
Should this be something like "If the grammar had conflicts, return a tuple which allows users to see what they were; or `None` if there were no conflicts"?